chess-AI/experimental-work/train.py

```diff
@@ -0,0 +1,81 @@
+import random
+import chess
+from mcts import Mcts
+from nn_layout import PlayNetwork
+from output_representation import PlayNetworkPolicyConverter
+
+class MctsTrain: # algorithm for training the model.
+
+    def __init__(self,mcts_simulations):
+
+        # Set the starting fen_string
+        self.start_fen = ("rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP"
+        "/RNBQKBNR w KQkq - 0 1")
+
+        # Set the number of Monte Carlo Simulations
+        self.mcts_simulations = mcts_simulations
+        self.training_examples = []
+
+    def training_episode(self,nnet):
+
+        # At each episode need to set the training example to empty
+        self.training_examples = []
+
+        # The board needs to be initialized ot the starting state
+        board = chess.Board(self.start_fen)
+        fen_string = self.start_fen
+        mcts = Mcts() # assign MCTS object
+        policy_converter = PlayNetworkPolicyConverter()
+
+        while True:
+
```

Comment:
Not sure about convention, but I usually don't see spaces after while loop/for loop/function/conditional statements. Usually just
```python
for blah:
    first_line()
```
For keeping code consistent maybe we can all follow that if you're ok with that!